core/src/components/datetime-button/test/disabled/datetime-button.e2e.ts

```diff
@@ -0,0 +1,20 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('datetime-button: disabled buttons', () => {
```

Comment:
Should we have a screenshot test here to verify the `disabled` styling?

Reply:
Added